ui/js/controllers/perf/graphs.js

```diff
@@ -45,7 +45,6 @@ perf.controller('GraphsCtrl', [
         function deselectDataPoint() {
             $timeout(function() {
                 $scope.selectedDataPoint = null;
-                $scope.plot.unhighlight();
```

Comment:
Actually, this line been added in another closed PR( https://github.com/mozilla/treeherder/pull/951/files#diff-30e3c5e057bc86a56c89530b58b3be63R48.)  Obviously, it's unnecessary.